tests/cisco_ios/show_alert_counters/cisco_ios_show_alert_counters.raw

```diff
@@ -0,0 +1,55 @@
+Interface: Fa1/1
+Error Code: ALERT-5-STP_BLOCK
+Timestamp:  2020 04 28 19 12 03
+Description:  This port is blocked by Spanning Tree Protocol (STP).
+Recommendation:  Make sure that no Spanning Tree Protocol loops exist in the network.
+
+
+Interface: Fa1/2
+Error Code: ALERT-5-STP_BLOCK
+Timestamp:  2020 08 10 11 34 47
+Description:  This port is blocked by Spanning Tree Protocol (STP).
+Recommendation:  Make sure that no Spanning Tree Protocol loops exist in the network.
+
+
+Error Code: ALERT-5-COLLISION_ERR
+Timestamp:  2020 08 27 13 50 25
+Description:  Detected lower performance on this link, possibly because of half duplex configuration or duplex mismatch.
+Recommendation:  Change the duplex setting of both the ports on this link to "auto" or "full-duplex".
+
+
+Interface: Fa1/3
+Error Code: ALERT-4-PORT_PSECURE_VIOLATION
+Timestamp:  2020 08 27 13 53 22
+Description:  Access denied to one or more connecting devices on this port.
+Recommendation:  Maximum allowed devices on this port are already connected, or an unauthorized device attempted to connect on this secure port. Disconnect the device.
+
+
+Error Code: ALERT-5-ALIGN_FCS_ERR
+Timestamp:  2020 08 27 13 48 25
+Description:  Detected a faulty cable, or bad hardware on connected device, or the connected device is generating frames with bad Frame Check Sequence (FCS).
+Recommendation:  Change the duplex setting of both the ports on this link to "full-duplex" or change the duplex setting of this port to "auto". Check if the cable is faulty and replace it if needed. The hardware on the connected device might be bad, disconnect the connected device.
+
+
+Error Code: ALERT-5-RX_TX_ERR
+Timestamp:  2020 08 27 13 48 25
+Description:  Detected a possible over-utilization of the switch link or bad frames generated by the connected device.
+Recommendation:  Change the port settings of this port to "auto". It might help to create an Ether Channel by grouping this port with other under-utilized or unused ports. Disconnect the connected device.
+
+
+Interface: Fa1/4
+
+Interface: Fa1/5
+
+Interface: Fa1/6
+
+Interface: Fa1/7
+
+Interface: Fa1/8
+
+Interface: Gi1/1
+
```

Comment:
This template should still capture all the interfaces.

Look into the `-> Continue.Record` option to capture the same output you have plus the additional blank interfaces.